### PR TITLE
Add refills to medication

### PR DESCRIPTION
**Pull request by mayerm94**
Add refills support to medication datatype to align with model info.
Pull requests into Health Data Standards require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.
 
**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1460
- [x] Internal ticket links back to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] Tests are included and test edge cases NA
- [x] Tests have been run locally and pass
- [x] Code coverage has not gone down and all code touched or added is covered.
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here:
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here:
 
**Cypress Reviewer:**
 
Name: @okeefm
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases (in cql_qdm_patientapi)
- [x] You have tried to break the code
 
**Bonnie Reviewer:**
 
Name: @hossenlopp
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: lib/health-data-standards/models/medication.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ class Medication < Entry
   # Bonnie displays this as 'dosage'.
   field :dose, type: Hash
   field :supply, type: Hash
+  field :refills, type: Hash
```
Comment (hossenlopp) [inline lib/health-data-standards/models/medication.rb:9]: I'd add a comment above `:administrationTiming` that notes it is equivalent to QDM 5.3 'frequency'.

**Review by hossenlopp — CHANGES_REQUESTED** [review]
Small suggestion.

File: lib/health-data-standards/models/medication.rb (hunk 1 above)
Comment (okeefm) [reply lib/health-data-standards/models/medication.rb:9]: What does the `Hash` for `:refills` look like?

Comment (mayerm94) [reply lib/health-data-standards/models/medication.rb:9]: 'refills': {unit: '', scalar: 5} ends up being a cql.Quantity in BONNIE. Tested here https://github.com/projecttacoma/cql_qdm_patientapi/blob/05e24c98bd724681026d60e76630452f71a31c81/spec/javascripts/datatypes/medication_spec.js.coffee

**Review by okeefm — APPROVED** [review]
Looks good to me.

**Review by hossenlopp — APPROVED** [review]
